automation/jenkins_build.sh

```diff
@@ -297,16 +297,18 @@ deploy_to_s3() {
 
 	deploy_build "$_s3_deploy_images_dir" "false"
 
```

Comment:
I think it would easier and cleaner to make the bucket a function parameter and just call the function (deploy_to_s3) twice.

Reply:
That's what I propose. As in this way we can easily remove this in the future - we will stop deploying in two places hopefully soon.